### ISSUE TITLE
Test removing the normilization step from lens training

### ISSUE THREAD

**Issue by levmckinney** [AlignmentResearch/tuned-lens#22]
This function in the training loop was added to handle the fact that the layer norm applied before the unmembering layer creates a situation where multiply possible affine transformations have the same effect.

https://github.com/AlignmentResearch/tuned-lens/blob/61a6ed89ad099456d9dcbf1126b4145850a107eb/tuned_lens/scripts/train_loop.py#L246

However, now that we are using weight decay this is probably no longer necessary.

**Comment by levmckinney** [comment]
Done it does not seem to do much https://wandb.ai/levmckinney/tuned-lens/runs/jpayadfz?workspace=user-levmckinney and https://wandb.ai/levmckinney/uncategorized/runs/a5k17p4d?workspace=user-levmckinney
